src/common/components/menu/MenuGrowthExercise.tsx

```diff
@@ -0,0 +1,134 @@
+import { Box, Button, Flex, HStack, VStack, Text } from "@chakra-ui/react"
+import { GuildIcon } from "common/components/GuildIcon";
+import { UserAvatar } from "./UserAvatar"
+import { useState } from "react";
+import { TGuild } from "types";
+// @ts-expect-error: Svg import
+import StudyExerciseIcon from "assets/StudyExerciseIcon.svg?react";
+// @ts-expect-error: Svg import
+import BlogArticleIcon from "assets/BlogArticleIcon.svg?react";
+// @ts-expect-error: Svg import
+import TodayILearnedIcon from "assets/TodayILearnedIcon.svg?react";
+
+type MenuGrowthExerciseProps = {
+    guild?: TGuild;
+    handleCreateGE: (type: "study" | "blog-article" | "til") => void;
+    title: string;
+}
+
+export const MenuGrowthExercise:React.FC<MenuGrowthExerciseProps> = ({guild, handleCreateGE, title}) => {
```

Comment:
Is this a view? The name isn't really conveying what the component is supposed to do can we come up with a better name?